testing/python/language/test_tilelang_language_dialect.py

```
@@ -0,0 +1,99 @@
+
+
+def test_default_language_dialect_env_controls_from_tilelang_import_language():
+    script = """
+from tilelang import language as T
+print("dialect", getattr(T, "__tilelang_dialect__", "missing"))
+print("module", T.__name__)
+print("has_tcgen05_mma", hasattr(T, "tcgen05_mma"))
+"""
+    env = dict(os.environ)
+    env["TILELANG_DEFAULT_DIALECT"] = "cuda"
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)